aircast/rootfs/usr/bin/shairport-play-handler.py

```
@@ -0,0 +1,230 @@
+        
+        self.ffmpeg_process = subprocess.Popen(
+            ffmpeg_cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE
+        )
+        
+        # Start simple HTTP server using Python
+        http_cmd = [
+            'python3', '-m', 'http.server',
+            str(port),
+            '--directory', '/tmp',
```

Ruff (0.14.7)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)